presto-docs/src/main/sphinx/sql/show-stats.rst

```diff
@@ -0,0 +1,30 @@
+==========
+SHOW STATS
+==========
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    SHOW STATS FOR table [ WHERE condition ]
+
+Description
+-----------
+
+Returns stats for the results of the provided query.
+
+Stats are returned for each column, along with a rollup across all columns
+(the column will be ``NULL`` for this rollup).
+
+=====================   ============
+Stat                    Description
```

Comment:
Let's rename `Stat` to `Column` here, since we are describing the results of the columns from the `SHOW STATS` query.

Reply:
I can do this, however I was trying to avoid the confusion that might be caused since these are for each column of the results.